src/entities/reading_tip.py

```diff
@@ -19,18 +19,10 @@ def format(self, seperator=", ") -> str:
         """Formats the reading tip values to a string using the specified seperator (default ", ").
         """
         values = []
-        if self.id:
-            values.append(f"Id: {self.id}")
-        if self.title:
-            values.append(f"Title: {self.title}")
-        if self.type:
-            values.append(f"Type: {self.type}")
-        if self.author:
-            values.append(f"Author: {self.author}")
-        if self.url:
-            values.append(f"URL: {self.url}")
-        if self.status:
-            values.append(f"Status: {self.status}")
+        attributes = {"Id": self.id, "Title": self.title, "Type": self.type, "Author": self.author, "URL": self.url, "Status": self.status}
+        for key, attr in attributes.items():
+            if attr:
+                values.append(": ".join([key, attr]))
```

Comment:
```suggestion
                values.append(": ".join([key, str(attr)]))
```
Not sure if this is just failing for the python version we use but this seems to fix the error.